lego/apps/companies/filters.py

```diff
@@ -17,3 +18,8 @@ class CompanyInterestFilterSet(FilterSet):
     class Meta:
         model = CompanyInterest
         fields = ("semesters", "events")
+
+
+class CompanyOrderingFilter(filters.OrderingFilter):
```

Comment:
```suggestion
class AdminCompanyFilterSet(FilterSet):
```

Reply:
Hmm I can't seem to get ordering to work with a `FilterSet` and I can't find any examples of it done in the codebase either. If you look at e.g. `events/views.py`, ordering is done using `rest_framework` `filters`, while normal filtering is done using `django_filters` `FilterSet`. Since the `AdminCompanyViewSet` does not have any normal filtering, wouldn't it make sense to revert it to not have the filters `companies/filters.py`?

Reply:
ah yes I was a bit quick here - I guess it makes sense to use the `rest_framework` `filters` since we're really only doing ordering here - sorry!